src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/AmqpConverterTests.cs

```diff
@@ -14,8 +14,8 @@ namespace Microsoft.Azure.ServiceBus.UnitTests
     public class AmqpConverterTests
     {
         [Fact]
-        [DisplayTestMethodName]        
-        void Convert_Amqp_message_with_Amqp_value_array_segment_to_SB_message()
+        [DisplayTestMethodName]
```

Comment:
If I'm not mistaken this attribute can be applied at the class level.

Reply:
Not sure if it's worth doing now but maybe something to clean up tests in the future.

Reply:
Doesn't look like it.   That's a custom attribute, based on one of the SDK extension points and looks to be allowing method targeting only:

```c#
[AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
public abstract class BeforeAfterTestAttribute : Attribute
```

Reply:
Honestly, I'm not sure what value it has at all and had thought about removing it.  It looks like we could hoist to the class level, but all that it is doing is spitting out diagnostic `Begin XXX` and `End XXX` methods to `Console.WriteLine` and `Debug.WriteLine`.    Personally, I think we get more value by just enabling the intrinsic XUnit diagnostics for insight into what the runner is up to.